build/docker/old/ubuntu-trusty/Dockerfile

```diff
@@ -190,10 +190,11 @@ RUN apt-get install -y --no-install-recommends \
       python3-six \
       python3-wheel \
       python3-zope.interface && \
-    pip install -U ipaddress backports.ssl_match_hostname tornado && \
-    pip3 install -U backports.ssl_match_hostname tornado
-# installing tornado by pip/pip3 instead of debian package
-# if we install the debian package, the build fails in py2
+      python3 -m pip install --upgrade pip && \
+    python3 -m pip install --upgrade \
+      ipaddress \
+      backports.ssl_match_hostname \
+      tornado
```

Comment:
the old comment seems to suggest installing tornado via debian package only affects python2, but I don't know if it will also affect python3.